Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Excel.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Data;
+using System.Globalization;
+using System.Text;
+using ExcelDataReader;
+
+namespace Frends.Excel.ConvertToCSV;
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to CSV. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.ConvertToCSV)
+    /// </summary>
+    /// <param name="input">Input configuration</param>
+    /// <param name="options">Input options</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns>Result containing the converted CSV string.</returns>
+    /// <exception cref="Exception"></exception>
+    public static Result ConvertToCSV(Input input, Options options, CancellationToken cancellationToken)
+    {
+        try
+        {
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            using (var stream = new FileStream(input.Path, FileMode.Open))
+            {
+                using (var excelReader = ExcelReaderFactory.CreateReader(stream))
+                {
+                    var result = excelReader.AsDataSet();
+                    var csv = ConveDataSetToCSV(result, options, cancellationToken);
+                    return new Result(true, csv, null);
+                }
+            }
+        }
+        catch (Exception ex)
+        {
+            if (options.ThrowErrorOnFailure)
+            {
+                throw new Exception("Error while converting Excel file to CSV", ex);
+            }
+
+            return new Result(false, null, $"Error while converting Excel file to CSV: {ex}");
+        }
+    }
+
+    /// <summary>
+    /// Converts DataSet-object to CSV.
+    /// </summary>
+    /// <param name="result">DataSet-object</param>
+    /// <param name="options">Input configurations</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    /// <returns>String containing the converted Excel.</returns>
+    private static string ConveDataSetToCSV(DataSet result, Options options, CancellationToken cancellationToken)
```

Comment:
Maybe add #region HelperMethods before class definition to indicate that the following methods are helper methods for the class to the developer.

Reply:
I will remove the overly spammy comments from those methods as they are pretty self-explanatory anyway and I think that private keywords would do the work otherwise to indicate the privacy / helperness level of the methods.